code/Whitespace/helloworld.ws

```diff
@@ -0,0 +1,34 @@
+   	      	 
+   			 		   		  	 
+    
+		    	  	   
+	
+     		  	 	
+	
+     		 		  
```

Comment:
Why so many unnecessary new lines? Remove them @unknown1924

Reply:
The above lines may seem empty but instead are filled with spaces and tabs.
This "invisible" code is written in whitespace [ https://tio.run/#whitespace ]
And produces "hello world" in output.
![image](https://user-images.githubusercontent.com/31164566/85734238-806ed000-b71a-11ea-9f0d-c5a9134606f0.png)

Reply:
I see. This is the Whitespace Lang but still it's not rendered my markdown. You have to insert text.

Reply:
okay! that's cool. I didn't know about this before.